tables/ru-litbrl.ctb

```diff
@@ -420,6 +511,10 @@ noback context _"$"[]%upperlatin @46#1=1#2=1                # lowercase latin le
 noback context _"$"[]%lowergreek @56#1=2#2=0                # lowercase greek letter following dollar sign: 56
 noback context _"$"[]%uppergreek @456#1=2#2=1               # uppercase greek letter following dollar sign: 456
 
+# Set the Cyrillic mode where it is required.
+noback context !#1=0!$d[]%lowercyrillic #1=0#2=0
+noback context !#1=0!$d[]%uppercyrillic #1=0#2=0
```

Comment:
I don't like this addition. I think it makes things more brittle. I tried removing the rules and adding the following two rules at the bottom, after the "Mark cyrillic letters immediately following latins and greeks" section:

```
# Enter cyrillic mode
noback context !#1=0[]%lowercyrillic #1=0#2=0
noback context !#1=0[]%uppercyrillic #1=0#2=0
```

and the tests still pass. What do you think?

Reply:
Hm, I wrote these rules like You, but I didn't gess to put them after the other ones. Yes, it works, I'll commit it soon.